tests/Aspire.Hosting.Azure.Tests/AzureServiceBusExtensionsTests.cs

```diff
@@ -633,4 +633,64 @@ public async Task AzureServiceBusEmulatorResourceGeneratesConfigJsonWithCustomiz
 
         await app.StopAsync();
     }
+
+    [Fact]
+    [RequiresDocker]
+    public async Task AzureServiceBusEmulator_WithConfigurationFile()
+    {
+        using var builder = TestDistributedApplicationBuilder.Create();
+
+        var configJsonPath = Path.GetTempFileName();
+
+        File.WriteAllText(configJsonPath, """
+            {
+              "UserConfig": {
+                "Namespaces": [
+                  {
+                    "Name": "servicebusns",
+                    "Queues": [ "queue456" ],
+                    "Topics": []
+                  }
+                ]
+              }
+            }
+            """);
+
+        var serviceBus = builder.AddAzureServiceBus("servicebusns")
+            .RunAsEmulator(configure => configure.WithConfigurationFile(configJsonPath));
+
+        using var app = builder.Build();
+        await app.StartAsync();
```

Comment:
Do we really need to start the application to do this test? Isn't it just verifying the annotation gets created correctly?